pkg/sbom/spdx/unmarshal.go

```diff
@@ -192,9 +205,9 @@ func parsePkg(spdxPkg spdx.Package) (*ftypes.Package, error) {
 			return nil, xerrors.Errorf("failed to parse source info: %w", err)
 		}
 	}
-	for _, f := range spdxPkg.Files {
-		pkg.FilePath = f.FileName
-		break // Take the first file name
```

Comment:
We probably should keep this logic for a while so that Trivy can parse old SPDX files.

Reply:
We don't need that. 
`tools-golang` converts `hasfiles` to `relationship` - https://github.com/spdx/tools-golang/blob/e939624e362e42cbd362b1d4ecaf1167a42d4901/spdx/v2/v2_2/document.go#L84 

I tested this case. 
Take a look on this testcase - https://github.com/aquasecurity/trivy/pull/4513/files#diff-6a7a7855bcf6bb4b61c399fb4a61a11959c089f7669a0785117f98b8efc25e89R60

Reply:
Oh, nice. It's ok, then.